### PR TITLE
use github templating mechanisms to classify issues when they are created

### PR DESCRIPTION
**Pull request by paulfantom**
##### Summary
Instead of relying on travis, use github to assign labels to new issues.

##### Component Name
ci, github

##### Additional Information
none

### CODE REVIEW

**Comment by paulkatsoulakis** [comment]
Hello, is this linked with an issue describing the details and how we ended up with the proposed PR?

**Comment by paulfantom** [comment]
> Hello, is this linked with an issue describing the details and how we ended up with the proposed PR?

No, there is no documented requirement in CONTRIBUTING.md to link PRs to anything.

**Comment by paulkatsoulakis** [comment]
> > Hello, is this linked with an issue describing the details and how we ended up with the proposed PR?
> 
> No, there is no documented requirement in CONTRIBUTING.md to link PRs to anything.

Ok, so there is no issue open, thanks. Just to clarify, the question was whether there is an issue linked to this PR, not "why you opened a PR"

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+about: General issue template
+labels: "needs triage", "no changelog"
```
Comment (paulkatsoulakis) [inline .github/ISSUE_TEMPLATE.md]: That should be "question", "no changelog"

**Pull request merged**